src/quo2/screens/user_avatar.cljs

```diff
@@ -0,0 +1,55 @@
+(ns quo2.screens.user-avatar
+  (:require [quo.react-native :as rn]
+            [quo.design-system.colors :as colors]
+            [quo.previews.preview :as preview]
+            [quo2.components.user-avatar :as quo2]
+            [reagent.core :as reagent]))
+
+(def descriptor [{:label   "Size:"
+                  :key     :size
+                  :type    :select
+                  :options [{:key   :big
+                             :value "Big"}
+                            {:key   :medium
+                             :value "Medium"}
+                            {:key   :small
+                             :value "Small"}]}
+                 {:label "Online status"
+                  :key   :online?
+                  :type  :boolean}
+                 {:label "Status Indicator"
+                  :key   :status-indicator?
+                  :type  :boolean}
+                 {:label "Identicon Ring"
+                  :key   :ring?
+                  :type  :boolean}
+                 {:label "Full name seperated by space"
```

Comment:
nit: I think you have a typo here. Should be sep'a'rated

Reply:
You're totally right, Thanks!